Ginger/GingerCoreNET/WorkSpaceLib/Solution.cs

```diff
@@ -588,6 +566,69 @@ public ePlatformType GetApplicationPlatformForTargetApp(string targetapp)
             return ePlatformType.NA;
         }
 
+        public override bool SerializationError(SerializationErrorType errorType, string name, string value)
+        {
```

Comment:
@Arvind, please add comments explaining why this is needed. So later if someone else will get to this code he will understand

Reply:
Done please review